user/user-application/src/main/java/me/nalab/user/application/common/dto/LoginedInfo.java

```diff
@@ -1,12 +1,15 @@
 package me.nalab.user.application.common.dto;
 
-import lombok.Data;
-
-@Data
-public class LoginedInfo {
-
-	private final String nickName;
-	private final Long targetId;
-	private final Long userId;
-
+import me.nalab.user.domain.user.User;
+
+public record LoginedInfo(
+    Long id,
+    Long targetId,
+    String nickname,
+    String email
+) {
+
+    public static LoginedInfo from(Long targetId, User user) {
```

Comment:
parameter가 복수개라면 [of](https://tecoble.techcourse.co.kr/post/2020-05-26-static-factory-method/)?
```suggestion
    public static LoginedInfo of(Long targetId, User user) {
```
<img width="690" alt="image" src="https://github.com/depromeet/na-lab-server/assets/61923768/b14fbf19-4d97-43db-8145-eafea715b77a">